ribs/archives/_cvt_archive.py

```diff
@@ -157,6 +159,8 @@ def __init__(self,
         self._centroid_kd_tree = None
         self._ckdtree_kwargs = ({} if ckdtree_kwargs is None else
                                 ckdtree_kwargs.copy())
+        self.chunk_size = chunk_size
```

Comment:
"Private" attributes should be named, for example, `self._chuck_size`

Reply:
*chunk